src/reason_parser.mly

```diff
@@ -637,11 +631,11 @@ let mkcf_attrs ?(loc=dummy_loc()) d attrs =
 let mkctf_attrs d attrs =
   Ctf.mk ~attrs d
 
-let add_nonrec rf attrs pos =
+let add_nonrec rf attrs start_pos end_pos =
```

Comment:
There is a data structure called "loc" which encapsulate a object with it's location. 

If we can convert "rf" to "loc", we will be able to say
```
let add_nonrec rf attrs =
match rf.txt with 
...

let name = { txt = "nonrec"; loc = rf.loc }
...
```
which I found could make the code more concise.

I made a small utility to create a "loc" object, see comments below.